repository.py

```diff
@@ -791,28 +791,36 @@ def installFromYum(targets, mounts, progress_callback, cachedir):
         count = 0
         total = 0
         verify_count = 0
+        progressLine = re.compile('.*(\d+)/(\d+)$')
         while True:
             line = p.stdout.readline()
             if not line:
                 break
             line = line.rstrip()
             logger.log("DNF: %s" % line)
+            norm_line = ' '.join(line.split())
             if line == 'Resolving Dependencies':
                 progress_callback(1)
-            elif line == 'Dependencies Resolved':
+            elif line == 'Dependencies Resolved' or line == 'Dependencies resolved.':
```

Comment:
Does DNF really use both of these or can we just check for the new one?

Reply:
More compatibility code (support both).